src/K8sJanitor.WebApi/Repositories/Kubernetes/RoleRepository.cs

```diff
@@ -212,6 +213,36 @@ public async Task<string> CreateNamespaceFullAccessRole(string namespaceName)
                             "*"
                         }
                     },
+                    new V1PolicyRule
+                    {
+                        ApiGroups = new List<string>
+                        {
+                            "external-secrets.io"
+                        },
+                        Resources = new List<string>
+                        {
+                            "*"
```

Comment:
Maybe we should prevent the cluster scoped resources?

Reply:
These are not cluster scoped resources, but is added to the role in each namespace.
Or maybe I misunderstand you comment?

Reply:
The resources "*" rule will allow user provisioning of ClusterSecretStore and ClusterExternalSecret is what I mean. Perhaps we should not allow those, or any other cluster-scoped resource from the external-secrets.io API if there are any

Reply:
Done